tools/profile-endpoints.py

```diff
@@ -0,0 +1,355 @@
+import subprocess
+import requests
+import time
+
+# Constants
+UUID = "c672c3c9-ab54-4e19-827c-f2af329da814"
+DATAREPO_URL = "https://jade.datarepo-dev.broadinstitute.org"
+
+
+def run_command(command):
```

Comment:
The indents here don't seem to match PEP8 suggestions. I would recommend installing `black` (`pip install black`) and running it on this file to fix the indents.

Reply:
Ok sounds good! I used my terminal to run this ..

Reply:
I ran, should be a lot cleaner!